test/runtests.jl

```diff
@@ -287,6 +287,13 @@ end
     @test X′ == X
     # test for presence of superfluous wrapping
     @test typeof(X′) == typeof(X)
+
+    # Treating vectors as 1-column tables:
+    x = [1,2,3,4,5]
+    tbl = Tables.Columns(Tables.table(x))
```

Comment:
also `Tables.table` with the second positional argument would be good to test.

Reply:
What is the second positional argumnet?  I only see one method of `table`, with only a single positional argument.  Do you mean the `header` keyword argument?

Reply:
Ah - right. I meant `header`.

Sorry for mishaps (@quinnj knows the Tables.jl best so he probably would not make the mistakes I make, but I know he currently needs some help in handling the requests so I thought that I comment to move things forward a bit before he can get to it).

Reply:
Added a `header` test.